microlith/html/styles.css

```diff
@@ -0,0 +1 @@
lay:none}.contributor-list-box.show{visibility:visible}.contributor-list-box span{font-size:.875rem;line-height:1;color:#52566c;position:relative;padding-right:15px}.contributor-list-box .last-commit-date:after{content:"";width:1px;height:100%;background:#52566c;position:absolute;right:0;top:0}#otherContributor{margin-left:5px}.contributor-list-box ul{display:-webkit-box;display:-ms-flexbox;display:flex;-ms-flex-wrap:wrap;flex-wrap:wrap;padding-left:10px}.contributor-list-box ul:before{content:none}.contributor-list-box ul li{margin-right:10px;width:22px;height:22px;list-style:none}.contributor-list-box ul li:last-child{margin-right:0}.contributor-list-box ul li img{max-width:100%;border-radius:50%}@media only screen and (max-width:767px){.contributor-list-box span{padding-right:10px}#otherContributor{padding-right:0}}.action-panel{display:none;position:absolute;top:0;bottom:0;left:50%;width:50%;background-color:#f1f7fe;-webkit-box-orient:vertical;-webkit-box-direction:normal;-ms-flex-direction:column;flex-direction:column}.action-panel .run-code-output{border:0;-webkit-box-flex:1;-ms-flex-positive:1;flex-grow:1}.terminal-launched,.terminal-launched body{overflow:hidden}.terminal-launched .action-panel{display:-webkit-box;display:-ms-flexbox;display:flex}.terminal-launched .left-sidebar,.terminal-launched .toc.sidebar,.terminal-launched footer{display:none}.action-panel .controls{display:-webkit-box;display:-ms-flexbox;display:flex;background:#fff;border-bottom:1px solid #e5e5e5}.action-panel .control-button,.action-panel .control-label{padding:4px 10px;font-size:14px;line-height:20px;color:#73768a}.action-panel .control-button{border:0;background:transparent}.action-panel .control-label{-webkit-box-flex:1;-ms-flex-positive:1;flex-grow:1}.action-panel .controls>*+*{border-left:1px solid #e5e5e5}.action-panel .close{display:-webkit-box;display:-ms-flexbox;display:flex;-webkit-box-align:center;-ms-flex-align:center;align-items:center}.action-panel .close svg{margin-right:10px}@media (min-width:768px){.terminal-launched .body.container{position:relative}.terminal-launched main.article{width:50%;position:absolute;overflow-x:hidden;overflow-y:scroll;top:0;bottom:0;left:0;padding-right:2.5rem;padding-left:2.5rem}}.doc a[href^="http://"]:not([href*=".couchbase.com"]):not([href*=localhost]):not([href*="127.0.0.1"]):not(.remove-ext-icon):after,.doc a[href^="https://"]:not([href*=".couchbase.com"]):not([href*=localhost]):not([href*="127.0.0.1"]):not(.remove-ext-icon):after,a.add-ext-icon:after{font-family:Font Awesome Light,serif;content:"\f35d";font-size:inherit;color:inherit;display:none}.doc a[href^="http://"]:not([href*=".couchbase.com"]):not([href*=localhost]):not([href*="127.0.0.1"]):not(.remove-ext-icon)>.svg-inline--fa,.doc a[href^="https://"]:not([href*=".couchbase.com"]):not([href*=localhost]):not([href*="127.0.0.1"]):not(.remove-ext-icon)>.svg-inline--fa,a.add-ext-icon>.svg-inline--fa{margin-left:.25em}.no-callouts .conum{display:none!important}
```

Comment:
Should we include the un-minified version of the CSS? It'll make it easier to edit - at the cost of some extra bytes, but CMOS will likely be deployed on an internal network where it's less of a concern.

Also, the font references link out to fonts.googleapis.com, which won't work in an air-gap (and could possibly be a privacy concern as every page load makes a request to Google).

Reply:
Looks like we use Source Sans Pro, Source Code Pro, and Open Sans - first two are licensed under the [SIL Open Font License](https://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&id=OFL) and the latter is Apache-2.0, so there shouldn't be a licensing concern for just bundling them.